pkg/provisioningv2/rke2/planner/planner.go

```diff
@@ -950,8 +951,8 @@ func noRole(entry *planEntry) bool {
 	return !isEtcd(entry) && !isControlPlane(entry) && !isWorker(entry)
 }
 
-func anyRole(_ *planEntry) bool {
-	return true
+func anyRole(entry *planEntry) bool {
```

Comment:
why change this?

Reply:
I was think that `anyRole` means that the node has a role, but we don't care.

Reply:
When I made this the intent was for the results to be unfiltered. given the possible scenarios these will behave the same way.